src/main.js

```diff
@@ -36,39 +36,39 @@ const JSON5 = require('./parsers/json5')
 const md5Function = require('./functions/md5')
 
 /* Utility/helpers */
-const cleanVariable = require('./utils/cleanVariable')
-const appendDeepVariable = require('./utils/appendDeepVariable')
-const isValidValue = require('./utils/isValidValue')
-const PromiseTracker = require('./utils/PromiseTracker')
-const handleSignalEvents = require('./utils/handleSignalEvents')
-const formatFunctionArgs = require('./utils/formatFunctionArgs')
-const { trimSurroundingQuotes, ensureQuote, isSurroundedByQuotes, startsWithQuotedPipe } = require('./utils/quoteUtils')
-const deepLog = require('./utils/deep-log')
-const { splitByComma } = require('./utils/splitByComma')
+const cleanVariable = require('./utils/variables/cleanVariable')
+const appendDeepVariable = require('./utils/variables/appendDeepVariable')
+const isValidValue = require('./utils/validation/isValidValue')
+const PromiseTracker = require('./utils/system/PromiseTracker')
```

Comment:
i don't like the /system dir

keep anything you tucked in that dir in the root utils dir please